templates/spring-kotlin/files/src/main/kotlin/sh/platform/template/spring/kotlin/HelloController.kt

```diff
@@ -0,0 +1,13 @@
+package sh.platform.template.spring.kotlin
+
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class HelloController() {
+
+    @GetMapping("/")
+    fun helloKotlin(): String {
+        return "hello world from Platform.sh"
+    }
```

Comment:
Kotlin looks an awful lot like normal Java to me...

Reply:
yeap.